README.md

```diff
@@ -659,20 +671,54 @@ docker ps -a -f ancestor=ubuntu
 
 ## Contributing
 
+Just fork this repo and send pull requests.
```

Comment:
People who don't know Git or Github are terrified of pull requests.   That's why all the pictures.  I'm fine adding the below section if you can say you can edit using workflow or edit using Github, but there's no "just" involved.